src/wordpress.js

```diff
@@ -6,11 +6,46 @@ const fetch = require('node-fetch');
 const flatcache = require('flat-cache');
 const { DateTime } = require('luxon');
 
+const AMO_BLOG_BASE_URL = 'https://addons.mozilla.org/blog';
+const DEFAULT_WORDPRESS_BASE_URL = 'https://mozamo.wpengine.com';
+
 async function getNumPages(endPoint) {
   const result = await fetch(endPoint, { method: 'HEAD' });
   return result.headers.get('x-wp-totalpages') || 1;
 }
 
+const fixInternalURLs = (content) => {
+  return content.replace(
+    /https?:\/\/mozamo\.wpengine\.com/g,
```

Comment:
Did you see if changing the site URL changes the output? I guess it'll be worth having this anyway, but that might save on the replacements? It's on general settings.


<img width="1105" alt="General_Settings_‹_Mozilla_Add-ons_Blog_—_WordPress" src="https://user-images.githubusercontent.com/1514/112811994-83656b00-9074-11eb-8fe3-24f8877c5a1f.png">

Reply:
@muffinresearch please see https://github.com/mozilla/addons-blog/issues/9#issuecomment-808518332, I tried but this setting is used in the editor too...

Reply:
Sorry I missed that.